### PR TITLE
Wordsprocessing DocFormatProvider

### PR DESCRIPTION
**Pull request by velikovmartin**

### CODE REVIEW

File: libraries/radwordsprocessing/cross-platform.md [hunk 1]
```diff
@@ -23,6 +23,9 @@ In order to use the model of the **RadWordsProcessing** library in your cross-pl
 * **Telerik.Documents.Flow.dll**
 * **Telerik.Zip.dll**
 
+If you need to import DOC files, you will need to refer the following assembly:
```
Comment (TanyaDimitrova) [inline libraries/radwordsprocessing/cross-platform.md]: This section would be easier for the users to read if we add the information in a table

File: libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/doc.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (TanyaDimitrova) [inline libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/doc.md]: Remove this topic

File: libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+description: Import from Word 97 - Word 2003 Binary file format Using DocFormatProvider
+slug: radwordsprocessing-formats-and-conversion-doc-docformatprovider
+tags: using,docformatprovider
+published: True
+position: 2
+---
+
+# Using DocFormatProvider
+
+**DocFormatProvider** makes it easy to import a **RadFlowDocument** from DOC format, preserving the entire document structure and formatting.
```
Comment (TanyaDimitrova) [inline libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md]: **DocFormatProvider** makes it easy to import a DOC file into a **RadFlowDocument**, preserving the entire document structure and formatting.

File: libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+All you have to do in order to use **DocFormatProvider** is add references to the assemblies listed below:
```
Comment (TanyaDimitrova) [inline libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md:23]: To use **DocFormatProvider**, you should add references to the assemblies listed below:

File: libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+      
+* Telerik.Windows.Documents.Core.dll
+* Telerik.Windows.Documents.Flow.dll         
+* Telerik.Windows.Documents.Flow.FormatProvider.Doc
+* Telerik.Windows.Zip.dll
+          
+
+## Import
+
+In order to import a doc document, you need to use the **Import()** method of **DocFormatProvider**.
```
Comment (TanyaDimitrova) [inline libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md]: To import a DOC file, you should use the **Import()** method of **DocFormatProvider**.

File: libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+The code in **Example 1** shows how to use **DocFormatProvider** to import a doc document from a file.
```
Comment (TanyaDimitrova) [inline libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md]: This sentence seems unnecessary. Maybe we can delete it?

File: libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+	RadFlowDocument document;
+	using (Stream input = File.OpenRead("Sample.doc"))
+	{
+		DocFormatProvider provider = new DocFormatProvider();
+		document = provider.Import(input);
+	}
+{{endregion}}
+
+
+And here is how you can import a document from byte array containing the doc document:
```
Comment (TanyaDimitrova) [inline libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md]: And here is how you can import a document from a byte array containing the document:
or
And here is how you can import a document from a byte array containing the .doc file:

File: libraries/radwordsprocessing/formats-and-conversion/word-file-formats/docx/settings.md [hunk 8]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Settings
 page_title: Settings
-description: Settings
+description: This article describes the import/export settings which provide additional modification options
```
Comment (TanyaDimitrova) [inline libraries/radwordsprocessing/formats-and-conversion/word-file-formats/docx/settings.md:4]: This article describes the settings that you can use to control how the DOCX documents are imported/exported in WordsProcessing.

File: web.config [hunk 9]
```diff
@@ -148,6 +148,13 @@
 					</conditions>
 					<action type="Redirect" url="libraries/radspreadprocessing/formats-and-conversion/import-and-export-to-excel-file-formats/xlsx/{R:1}"/>
 				</rule>
+				<rule name="docx" enabled="true" stopProcessing="true">
```
Comment (TanyaDimitrova) [inline web.config:151]: Verify that works after merging it

File: libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+position: 1
+---
+
+# Doc
+
+DOC is the default target Binary file format developed by Microsoft for representing documents in Word 97 - Word 2003 and is one of the supported formats by RadWordsProcessing.
+
+# Dot
+
+DOT is а Word 97 - Word 2003 Binary file format, developed by Microsoft for representing document templates and is one of the supported formats by RadWordsProcessing.
```
Comment (TanyaDimitrova) [inline libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md:17]: Sounds pretty similar to the sentence above. Let's reword it

File: libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+
+# Using DocFormatProvider
+
+**DocFormatProvider** makes it easy to import a **RadFlowDocument** from DOC and DOT format, preserving the entire document structure and formatting.
```
Comment (TanyaDimitrova) [inline libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md:21]: **DocFormatProvider** makes it easy to import DOC or DOT files into a **RadFlowDocument**, preserving the entire document structure and formatting.

File: libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+
+All you have to do in order to use **DocFormatProvider** is add references to the assemblies listed below:
+      
+* Telerik.Windows.Documents.Core.dll
+* Telerik.Windows.Documents.Flow.dll         
+* Telerik.Windows.Documents.Flow.FormatProvider.Doc
```
Comment (TanyaDimitrova) [inline libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md:27]: Telerik.Windows.Documents.Flow.FormatProvider.Doc.dll

File: libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md [hunk 13]
```diff
@@ -0,0 +1,60 @@
...
+* Telerik.Windows.Zip.dll
+          
+
+## Import
+
+In order to import a DOC or DOT document, you need to use the **Import()** method of **DocFormatProvider**.
```
Comment (TanyaDimitrova) [inline libraries/radwordsprocessing/formats-and-conversion/word-file-formats/doc/docformatprovider.md:33]: To import a DOC or DOT document, you should use the **Import()** method of **DocFormatProvider**.

File: libraries/radwordsprocessing/formats-and-conversion/word-file-formats/docx/docxformatprovider.md [hunk 14]
```diff
@@ -1,18 +1,22 @@
 ---
 title: Using DocxFormatProvider
 page_title: Using DocxFormatProvider
-description: Using DocxFormatProvider
+description: Import and Export to Docx File Format Using DocxFormatProvider
```
Comment (TanyaDimitrova) [inline libraries/radwordsprocessing/formats-and-conversion/word-file-formats/docx/docxformatprovider.md:4]: Learn how you can import and export documents from/to DOCX using WordsProcessing.

**Review by TanyaDimitrova — CHANGES_REQUESTED** [review]
You will need to also update:
- https://docs.telerik.com/devtools/document-processing/getting-started/getting-started
- https://docs.telerik.com/devtools/document-processing/introduction

**Review by johncosmos653 — APPROVED** [review]

**Pull request merged**